src/components/controls/hormones/Hormones.css

```diff
@@ -0,0 +1,67 @@
+.Hormone-Container {
```

Comment:
@azizmashkour, you're inconsistent with the capitalisation of the classes. Sometimes it's `Upper-Upper`, sometimes it's `Upper-lower`, sometimes it's `lower-lower`. You can choose what you prefer, but be consistent!